src/components/EditCoursePage/index.jsx

```diff
@@ -301,26 +302,27 @@ class EditCoursePage extends React.Component {
       courseStatuses.push(UNPUBLISHED);
     }
 
-    const minimalCourseRuns = course_runs && course_runs.map(courseRun => ({
-      key: courseRun.key,
-      start: courseRun.start,
-      end: courseRun.end,
-      expected_program_type: courseRun.expected_program_type,
-      expected_program_name: courseRun.expected_program_name,
-      go_live_date: courseRun.go_live_date,
-      min_effort: typeof courseRun.min_effort === 'number' ? String(courseRun.min_effort) : '',
-      max_effort: typeof courseRun.max_effort === 'number' ? String(courseRun.max_effort) : '',
-      pacing_type: courseRun.pacing_type,
-      content_language: courseRun.content_language,
-      transcript_languages: courseRun.transcript_languages,
-      weeks_to_complete: typeof courseRun.weeks_to_complete === 'number' ?
-        String(courseRun.weeks_to_complete) : '',
-      staff: courseRun.staff,
-      status: courseRun.status,
-      draft: courseRun.draft,
-      marketing_url: courseRun.marketing_url,
-      has_ofac_restrictions: courseRun.has_ofac_restrictions,
-    }));
+    const minimalCourseRuns = course_runs && course_runs.map(courseRun =>
+      ({
+        key: courseRun.key,
+        start: courseRun.start,
+        end: courseRun.end,
+        expected_program_type: courseRun.expected_program_type,
+        expected_program_name: courseRun.expected_program_name,
+        go_live_date: courseRun.go_live_date,
+        min_effort: typeof courseRun.min_effort === 'number' ? String(courseRun.min_effort) : '',
+        max_effort: typeof courseRun.max_effort === 'number' ? String(courseRun.max_effort) : '',
+        pacing_type: courseRun.pacing_type,
+        content_language: courseRun.content_language,
+        transcript_languages: courseRun.transcript_languages,
+        weeks_to_complete: typeof courseRun.weeks_to_complete === 'number' ?
+          String(courseRun.weeks_to_complete) : '',
```

Comment:
Know this isn't your change, but is there a reason we need to cast this to a string? And if it's not a number we just don't render anything? Our DB has this restricted to a smallint already so I'd be really surprised if we passed in something that wasn't a number to this.

Reply:
There was a reason...  @Dillon-Dumesnil would know.

Reply:
It has to do with the form being set to pristine. It is returned from discovery as an int, but the form treats it as a string so in order for it to match initialValues with currentFormValues, we cast it to a string. The default value is just `''`